src/main/java/lotto/StringAddCalculator.java

```diff
@@ -0,0 +1,43 @@
+package lotto;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+	private static String DEFAULT_DELIMITER = ",|:";
+
+	public static int splitAndSum(String inputText) {
+		inputText = Optional.ofNullable(inputText).map(String::valueOf).orElse("");
+		Matcher m = Pattern.compile("//(.)\n(.*)").matcher(inputText);
+		if (m.find()) {
+			return sum(m.group(2).split(m.group(1)));
+		}
+		return sum(inputText.split(DEFAULT_DELIMITER));
+	}
+
+	private static Integer sum(String[] tokens) {
```

Comment:
```suggestion
	private static int sum(String[] tokens) {
```
메서드 리턴값에 null이 생길 여지를 만들지 않는게 좋습니다
다른 메서드도 같이 수정해보아요